duneanaobj/StandardRecord/SRTrack.h

```diff
@@ -0,0 +1,33 @@
+////////////////////////////////////////////////////////////////////////
+/// \file    SRTrack.h
+/// \brief   Reconstructed track object
+/// \author  J. Wolcott <jwolcott@fnal.gov>
+/// \date    Sept. 2021
+////////////////////////////////////////////////////////////////////////
+#ifndef DUNEANAOBJ_SRTRACK_H
+#define DUNEANAOBJ_SRTRACK_H
+
+#include "duneanaobj/StandardRecord/SRVector3D.h"
+#include "duneanaobj/StandardRecord/SRParticleTruth.h"
+
+namespace caf
+{
+  class SRTrack
+  {
+    public:
+      SRVector3D start;      ///< Track 3D start point
+      SRVector3D end;        ///< Track 3D end point
+      SRVector3D end_dir;    ///< Unit vector representing estimate of track direction *taken from endpoint*
```

Comment:
Is there no `dir` - an estimate of the track's initial direction?

Reply:
not currently at least.  the enddir was necessary for track matching, but (end - start) is probably a good enough first pass for most other things?

Reply:
I guess I'm thinking that the track direction is actually one of the key observables for xsec measurements. And if this object gets re-used in the GAr where there is a magnetic field end-start can be a terrible estimate.

Reply:
I don't have any problem adding a field for it as a placeholder. We don't currently have anything to fill it with, but I guess that's not a huge issue

Reply:
ok, added. 282ea6d8